source/mesh_adaptor.template.h

```diff
@@ -344,19 +344,19 @@ namespace ryujin
 
     switch (marking_strategy_) {
     case MarkingStrategy::fixed_number: {
-      dealii::GridRefinement::refine_and_coarsen_fixed_number(
-          triangulation,
-          indicators_,
-          refinement_fraction_,
-          coarsening_fraction_);
+      dealii::parallel::distributed::GridRefinement::
+          refine_and_coarsen_fixed_number(triangulation,
+                                          indicators_,
+                                          refinement_fraction_,
+                                          coarsening_fraction_,
+                                          max_num_cells_);
     } break;
     case MarkingStrategy::fixed_fraction: {
-      dealii::GridRefinement::refine_and_coarsen_fixed_fraction(
-          triangulation,
-          indicators_,
-          refinement_fraction_,
-          coarsening_fraction_,
-          max_num_cells_);
+      dealii::parallel::distributed::GridRefinement::
+          refine_and_coarsen_fixed_fraction(triangulation,
+                                            indicators_,
+                                            refinement_fraction_,
+                                            coarsening_fraction_);
```

Comment:
We should have a look at implementing this variant in deal.II with a global `max_num_cells` limit as well.